### PR TITLE
e2e: validate images in trash

### PR DESCRIPTION
**Pull request by Madhu-1**
when all the PVCs and associated images are deleted, the images should also get deleted from the trash. This commit adds the validation check for the same.

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

**Comment by Madhu-1** [comment]
> /retest ci/centos/mini-e2e-helm/k8s-1.20

Monitor connection error
```
Jun 15 07:00:27.173: INFO: stdErr occurred: 2021-06-15T06:00:27.143+0000 7fe6709b3700 -1 monclient: get_auth_request but no auth handler is set up

Jun 15 07:00:27.173: FAIL: failed to list rbd images with error failed to list images in trash 2021-06-15T06:00:27.143+0000 7fe6709b3700 -1 monclient: get_auth_request but no auth handler is set up
```

**Comment by Madhu-1** [comment]
> /retest ci/centos/mini-e2e/k8s-1.20

This also failed with monitor auth error

File: e2e/rbd.go [hunk 1]
```diff
@@ -1250,6 +1250,15 @@ var _ = Describe("RBD", func() {
 				if err != nil {
 					e2elog.Failf("failed to delete snapshotclass with error %v", err)
 				}
+				// validate images in trash
+				err = waitToRemoveImagesFromTrash(f, clonePool, deployTimeout)
+				if err != nil {
+					e2elog.Failf("failed to validate rbd images with error %v", err)
```
Comment (nixpanic) [inline e2e/rbd.go]: better to include the name of the pool in the error message too

**Review by nixpanic — APPROVED** [review]

**Review by Rakshith-R — APPROVED** [review]

**Comment by nixpanic** [comment]
> /retest ci/centos/mini-e2e/k8s-1.20

Failed to start minikube VM:
```
* Failed to start kvm2 VM. Running "minikube delete" may fix it: creating host: create: Error creating machine: Error in driver during machine creation: IP not available after waiting: machine minikube didn't return IP after 1 minute


X Exiting due to GUEST_PROVISION: Failed to start host: creating host: create: Error creating machine: Error in driver during machine creation: IP not available after waiting: machine minikube didn't return IP after 1 minute
```

**Pull request merged**